examples/Cpp/AlarmClock/src/README.md

```diff
@@ -1,63 +1,28 @@
 Lingua Franca Alarm Clock
 ----------------------------
 
-**Contact:** <revol-xut@protonmail.com>
+**Author:** <revol-xut@protonmail.com>
 
-**Main Repository:** [](https://github.com/revol-xut/lf-alarm-clock)
-
-A small and tiny alarmclock which is written using the scheduling and time features from lingua franca. 
-
-## What you will learn
-
-- sharing state between reactors
-- stopping scheduled events
-
-## Project
+A small alarm clock which is written using the scheduling and time features from Lingua Franca.
+This requires [installing Crow](https://crowcpp.org/master/getting_started/setup), which provides the HTTP server implementation.
 
 ![Programm Structure](./images/entire_program.png)
 
-
-## Building
-
-**Dependencies:** jdk11, boost, mpg321, Crow
-
-
-```bash
-    $ lfc ./AlarmClock.lf
-```
-
-**Building with nix**
-
-This cross compiles for aarch64.
-```
-    nix build .#packages.aarch64-linux.lf-alarm-clock
-```
-
 ## Installation
 
 By default the AlarmClock expects the sound files to be placed in `~/music/AlarmClock/` you can change this
 path by editing the `shared_header.cpp` file. Furthermore is it possible to configure paths to other binaries
 in this file e.g. kill, mpg321 -commands. 
 
-### Installing Crow from source
-
-On most distros, Crow needs to be build and installed from source:
+If you have installed Crow in a location where CMake does not automatically find it, then you can manually specify the location when compiling the LF program as follows:
 
-```bash
-    $ git clone git@github.com:CrowCpp/Crow.git
-    $ mkdir Crow/build
-    $ cd Crow/build
```

Comment:
These instructions seem crucial for building the example on a local machine. Why are they removed? I didn't find a replacement for those instructions elsewhere.

Reply:
These instructions are platform dependent and incomplete (e.g., you have to also install asio). I have replaced them with a pointer to the Crow installation web page.